### ISSUE TITLE
Undo rename/copy/delete

### ISSUE THREAD

**Issue by eugenesvk** [fman-community/fman#209]
Is it possible to **undo** operations?
Also, when `delete to trash` option is on, is it possible to remove the confirmation dialogue? It's much faster and convenient (but not less safe) to undo an occasional mistaken delete (say, 10% of the time), rather than have an extra confirmation to perform 100% of the time.
Lack of this feature is the major drawback of many other "advanced" file managers. Hopefully fman will not be one of those :)

**Comment by mherrmann** [comment]
Undo is currently not implemented. I don't think I want to remove the extra confirmation for putting items to the trash. Most users simply expect it and if they don't know that undo is possible, they will believe fman deletes their files like a shotgun.

I don't know when / if undo will be implemented in core fman. (Depends on the number of votes this features gets). But undo is definitely possible to implement as a plugin with the current API. If you would like to write a plugin that does this, I'd be more than happy to help you.

**Comment by macosxguru** [comment]
Undo as a feature would be very welcome either as a core feature or as a plugin.

macosxguru

**Comment by eugenesvk** [comment]
The extra confirmation can still be the default for the "most users". I'm just asking to make it possible to disable it (as it is possible to disable this feature in other file managers, leaving confirmation only for permanent delete, which skips trash)
Unfortunately, I don't know enough to write such a plugin.

**Comment by mherrmann** [comment]
@eugenesvk have you had a look at the [tutorial for writing plugins](https://fman.io/docs/plugins-introduction)?

**Comment by eugenesvk** [comment]
@mherrmann yep, I have, and that's about as much I know programming to be able to write :)
Could also do a couple of formatting tweaks like so https://github.com/kek91/StatusBarExtended/pull/14, but the undo for different operations is way too complicated
